### PR TITLE
Fix build fail due to missing bitcode

### LINKED ISSUE #18: Xcode 7 builds fail due to missing bitcode
Attempting to build the Rollbar 0.1.5 pod for an iOS device using Xcode 7 fails:
```ld: '~/Library/Developer/Xcode/DerivedData/<...>/Build/Products/Debug-iphoneos/Rollbar.framework/Rollbar' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64```

Looks like resolving this requires an updated PLCrashReporter framework (see also https://opensource.plausible.coop/bugs/browse/PLCR-581).

Are you folks aware of this and working with Plausible Labs on a solution?

### PR DESCRIPTION
**Pull request by TigerWolf**
This fixes #18 

Thanks goes to: https://github.com/bitstadium/HockeySDK-iOS/commit/3d0185bdb0065cc59df5ebeccebc21ca194a42e5

### CODE REVIEW

**Comment by Crisfole** [comment]
:+1: 💯! We'll take a look and merge ASAP. Thanks so much.

**Comment by ezarowny** [comment]
@TigerWolf I'll be taking a look at this today.

File: Vendor/CrashReporter.framework/Versions/A/Headers/PLCrashNamespace.h [hunk 1]
```diff
@@ -35,16 +35,25 @@
  * This may be used to avoid symbol conflicts between multiple libraries
  * that may both incorporate PLCrashReporter.
  */
-// #define PLCRASHREPORTER_PREFIX AcmeCo
+#define PLCRASHREPORTER_PREFIX BIT
```
Comment (ezarowny) [inline Vendor/CrashReporter.framework/Versions/A/Headers/PLCrashNamespace.h]: Why change this? It seems fine on my end but I was curious.

**Comment by ezarowny** [comment]
Things are looking good on my end. What version of PLCrashReporter is this based off of?

File: Vendor/CrashReporter.framework/Versions/A/Headers/PLCrashNamespace.h (hunk 1 above)
Comment (ezarowny) [inline Vendor/CrashReporter.framework/Versions/A/Headers/PLCrashNamespace.h]: Ah, this is Bit Stadium's prefix.

**Pull request merged**